src/Appwrite/Platform/Workers/Mails.php

```diff
@@ -78,8 +78,9 @@ public function action(Message $message, Registry $register, Log $log): void
         $bodyTemplate = Template::fromFile($bodyTemplate);
         $bodyTemplate->setParam('{{body}}', $body, escapeHtml: false);
         foreach ($variables as $key => $value) {
-            // TODO: hotfix for redirect param
-            $bodyTemplate->setParam('{{' . $key . '}}', $value, escapeHtml: $key !== 'redirect');
+            // TODO: hotfix for html params
+            $htmlParams = [ 'redirect', 'agentDevice', 'agentClient', 'agentOs', 'phrase' ];
```

Comment:
why cant those be escaped?

I think all of those expect redirect should be scaped.

Reply:
They have boldness, so `<b>SOMETHING</b>`. Without it the email looks very plain, and I couldn't find better way to do it

Reply:
This is what it looks likewithout changes in my PR
![CleanShot 2024-01-31 at 10 05 57@2x](https://github.com/appwrite/appwrite/assets/19310830/91a094ab-48fd-40d2-baab-41f8fc3bf5df)